backend/src/main/resources/application-create-sample-data.yaml

```diff
@@ -66,40 +96,49 @@ simple-report:
   demo-users:
     default-user:
       identity:
-        username: bob@sample.com
+        username: bob@example.com
         first-name: Bob
         last-name: Bobberoo
       authorization:
         organization-external-id: ${SR_DEMO_USER_ORG:DIS_ORG}
-        granted-roles: ${SR_DEMO_USER_ROLE:ADMIN}
+        granted-roles: ${SR_DEMO_USER_ROLE:MEMBER, ADMIN}
     alternate-users:
       - identity:
           username: ruby@example.com
           first-name: Ruby
           middle-name: Raven
           last-name: Reynolds
+        authorization:
+          organization-external-id: DIS_ORG
+          facilities: 
+            - Testing Site
+            # TODO: uncomment once e2e tests can accommodate 2+ facilities
+            # - Injection Site
+          granted-roles: MEMBER, USER
       - identity:
-          username: ben@sample.com
+          username: ben@example.com
           first-name: Ben
           middle-name: Billy
           last-name: Barnes
           suffix: III
         authorization:
           organization-external-id: DIS_ORG
-          granted-roles: USER
+          facilities: 
+            - Testing Site
+          granted-roles: MEMBER
       - identity:
-          username: jamar@sample.com
+          username: jamar@example.com
```

Comment:
I think that the same PR or two where we're renovating demo auth and test identities is the right time to also make sure these emails match. I know I've faced a lot of pain in the past because I accidentally used the wrong domain for a test email address somewhere, and this reduces the chance of those mixups.